.github/workflows/scripts/eslint_disable_check.py

```
@@ -0,0 +1,155 @@
+    except FileNotFoundError:
+        print(f"File not found: {file_path}")
+        return False
+    except PermissionError:
+        print(f"Permission denied: {file_path}")
+        return False
+    except (IOError, OSError) as e:
+        print(f"Error reading file {file_path}: {e}")
+        return False
+
+
+def check_eslint(files_or_directories):
```

Ruff (0.8.2)

`check_eslint` is too complex (11 > 10)

(C901)